x-pack/plugins/canvas/public/components/text_style_picker/text_style_picker.js

```diff
@@ -64,6 +64,11 @@ export const TextStylePicker = ({
     ['underline']: Boolean(underline),
   };
 
+  if (!isNaN(size) && fontSizes.indexOf(Number(size)) === -1) {
```

Comment:
Is the `isNaN` check required here? Invalid values come in as `NaNpx` (another bug honestly), and then the dropdown still just shows 0. This isn't really a problem (it's edge case and is caused by another problem) and I'm not sure what else it would show, but thought I'd ask.

Reply:
Opened an issue for the `NaN` thing I mentioned: https://github.com/elastic/kibana/issues/27788